src/data/quality-assurance-quiz.ts

```diff
@@ -182,7 +182,61 @@ const qualityAssuranceQuiz = [
     Explanation:
       "A good QA engineer should understand the full life cycle of software products in order to effectively plan and test them. There are 7 phases of the Software Development Life Cycle",
     Link: "https://www.freecodecamp.org/news/how-to-become-a-quality-assurance-engineer-qa-engineer-career-guide/"
-  }
+  },
+  {
+      Question: "What is the primary purpose of a test plan in software quality assurance?",
+      Answer: "A test plan outlines the scope, approach, resources, and schedule for testing activities. Its primary purpose is to guide the testing process and ensure that all aspects of testing are systematically covered.",
+      Distractor1: "A test plan is used to document defects found during testing.",
+      Distractor2: "A test plan specifies the code changes needed to fix defects.",
+      Distractor3: "A test plan is only created after testing is complete.",
+      Explanation: "Test plans are essential for planning, managing, and executing testing efforts in a structured manner. They help teams achieve comprehensive test coverage and meet quality goals.",
+      Link: "https://www.guru99.com/test-plan.html"
+    },
+    {
+      Question: "What is the key difference between functional testing and non-functional testing?",
+      Answer: "Functional testing checks whether a software application performs its intended functions correctly, while non-functional testing focuses on aspects like performance, scalability, security, and usability.",
+      Distractor1: "Functional testing is automated, while non-functional testing is manual.",
+      Distractor2: "Functional testing is concerned with code quality, while non-functional testing checks user interfaces.",
+      Distractor3: "Functional testing verifies user documentation, while non-functional testing assesses data integrity.",
+      Explanation: "Understanding the distinction between functional and non-functional testing is crucial for designing comprehensive test strategies.",
+      Link: "https://www.softwaretestinghelp.com/types-of-software-testing/"
```

Comment:
we only accept links from freeCodeCamp news publication, freeCodeCamp YouTube channel, or wikipedia
Please change this link